### PR TITLE
feat: initial alpha support of QuestDB HTTP API

### PR DESCRIPTION
**Pull request by mpsq**

### CODE REVIEW

File: pkg/helpers/query.go [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+	r := regexp.MustCompile(`\$__timeFilter\((?P<Col>[[:alnum:]_]+)\)`)
+	res := r.FindStringSubmatch(query)
+
+	if len(res) > 1 {
+		var clause string
+		col := res[1]
+
+		clause += col
+		clause += " > '"
+		clause += q.TimeRange.From.UTC().Format("2006-01-02T15:04:05.000Z")
```
Comment (bluestreak01) [inline pkg/helpers/query.go:30]: timestamp is hardcoded?

Can we have a comment above the function with what is it expecting to be parsing?

**Review by bluestreak01 — CHANGES_REQUESTED** [review]

File: pkg/helpers/query.go (hunk 1 above)
Comment (mpsq) [reply pkg/helpers/query.go:30]: No that is how you specify a format in Go.

Comment (mpsq) [reply pkg/helpers/query.go:30]: To know more about this, you can check https://www.pauladamsmith.com/blog/2011/05/go_time.html

Comment (bluestreak01) [reply pkg/helpers/query.go:30]: :+1:

**Review by bluestreak01 — DISMISSED** [review]

**Pull request merged**